packages/diagrams/backend/sirius-components-collaborative-diagrams/src/main/resources/schema/diagram.graphqls

```diff
@@ -632,6 +633,7 @@ input NodeLayoutDataInput {
   position: PositionInput!
   size: SizeInput!
   resizedByUser: Boolean!
+  minimumWidth: Int!
```

Comment:
Here's something I don't understand (and probably a reason why I would need an ADR to understand things more clearly). If I'm not mistaken, by looking at the backend code:

- The frontend computes somewhere a minimumWidth and gives it to the backend using the `NodeLayoutDataInput`
- The backend gives it backend to the frontend using the `NodeLayoutData`

Why? Why does this computed minimumWidth has to be given backend and forth to the backend since it is entirely derived from existing pieces of information known by the frontend?

Reply:
I could not have set this value during the conversion of the diagram using the existing width values found in `NodeLayoutData`
For an `IconLabel` the value of the width stored in `NodeLayoutData` coincides with the width of the parent.
The minimum width is found only during the layout when applying `getBoundingClientRect` on the temporarily rendered label.
I'll write and ADR to further explain the issue.